webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2959,6 +2974,37 @@ var userOverride = {
             _this.phoneNumberValidation = function (e) {
                 e.target.value = e.target.value.match(/^([0-9]{0,15})/)[0];
             };
+
+            _this.createSelectFieldDropdown = function (
+                options,
+                selectElement
+            ) {
+                var dropDownOption = document.createElement('option');
+                dropDownOption.setAttribute('value', '');
+                dropDownOption.innerHTML =
+                    'Please select ' +
```

Comment:
use MCK_LABELS for 'Please select' so that customers can configure it if they wish to change the text